Modules/CodeTrek/Services/CodeTrekService.php

```diff
@@ -23,7 +23,30 @@ public function getCodeTrekApplicants($data = [])
         ->paginate(config('constants.pagination_size'))
         ->appends(request()->except('page'));
 
-        return ['applicants' => $applicants];
+        $applicantsData = CodeTrekApplicant::whereIn('status', ['active', 'inactive', 'completed'])
+        ->when($search, function ($query) use ($search) {
+            return $query->whereRaw("CONCAT(first_name, ' ', last_name) LIKE '%$search%'");
+        })
+        ->groupBy('status')
+        ->selectRaw('count(status) as total, status')
+        ->get();
+
+        $statusCounts = [];
+
+        $statusValues = ['active', 'inactive', 'completed'];
+        foreach ($statusValues as $status) {
+            $statusCounts[$status] = 0;
+        }
+
+        foreach ($applicantsData as $data) {
+            $statusCounts[$data->status] = $data->total;
+        }
```

Comment:
The the slug valuesof status from config.php which is inside `Modules/CodeTrek/config`
<img width="465" alt="image" src="https://github.com/ColoredCow/portal/assets/54376313/17f1df52-f40f-48ab-b290-977b68193d92">

Reply:
Here take the key names from config
```suggestion
        $statusCounts = [
            'active' => 0,
            'inactive' => 0,
            'completed' => 0
        ];
        foreach ($applicantsData as $data) {
            $statusCounts[$data->status] = $data->total;
        }
```